shpy

```diff
@@ -7,22 +7,33 @@ createSpy() {
     fi
 
     # shellcheck disable=SC2039
-    local val output error_output
+    local val output error_output stub_dir outputs_dir last_arg
+    for last_arg; do true; done
```

Comment:
`last_arg` is supposed to always be the spy name, I think this would be more understandable with a name like `spy_name`

Reply:
Good catch